agent/lib/ragent.js

```diff
@@ -384,7 +384,7 @@ Ragent.prototype.listen = function (options) {
 }
 
 Ragent.prototype.subscribe_to_addresses = function (env) {
-    var address_source = new AddressSource('foo', env);
+    var address_source = new AddressSource({name: 'foo', uuid: '1234'}, env);
```

Comment:
@grs To me it seems like env is actually options passed on to kubernetes.watch( in AddressSource() constructor, so I wasn't sure if that would break it. Wdyt?